script/Arcueid/Day 5/05_00_ARC05_10.txt

```diff
@@ -32,17 +33,24 @@ I'm not even sure if I'm still holding the knife.
 [sha:72dfe94f3d15499a23fb65824296f4b38e993c54]{
 -- Page 715, Offset 10598.
 -- 確かなものは全身の痛みと、気を緩めた途端、闇に落ちそうな意識だけ。
-The only things I can be sure of are the pain all over my body, and the fact that as soon as the tension was released my consciousness began to sink into the darkness.
+The only things I can be sure of are the pain all over my body,
+ and the fact that as soon as the tension was released,
+ my consciousness began to sink into the darkness.
 }
 [sha:86d048441ccef34081f90fc1e10cbaafde4295bf]{
 -- Page 716, Offset 10599.
 -- ……まったく幸いだ。おかげで痛覚も麻痺している。
-...I really am fortunate. My sense of pain has been numbed thanks to that.
+...I really am fortunate.
+ Thanks to that my sense of pain was numbed.
 }
 [sha:71bd54dd497802a9a2cfa62ff25d4d4158ba3a61]{
 -- Page 716, Offset 10600.
 -- 滑空中に踏み込んだ左足の痛みは平時なら耐えられるものじゃない。骨ごとノコギリで削られているようだ。そんな、目から火花が出るほどの痛みも、今はまともに感じない。
-The pain in my left foot, which I broke while gliding mid-air, would have been unbearable otherwise. It's as if every bone in my body is being sawed away. I can't even feel the pain that sparked from my eyes anymore.
+The pain in my left foot,
+ which I broke mid-air,
+ would have been normally been unbearable.
+ It's as if every bone in my body is being hacked off.
+ I can't even feel the pain that had me seeing sparks before.
```

Comment:
I prefer sparks because, and I'm sure this is some sort of nasuverse faux pas, but I'm thinking of Ryougi's eye effects in the KnK movies, which are sorta spark-like

Reply:
prefer stars but don't really mind other